sigal/__init__.py

```diff
@@ -117,8 +117,14 @@ def build(source, destination, debug, verbose, force, config, theme, title,
         logger.error("Input directory not found: %s", settings['source'])
         sys.exit(1)
 
-    if not os.path.relpath(settings['destination'],
-                           settings['source']).startswith('..'):
+    relative_check = True
+    try:
+        relative_check =  os.path.relpath(settings['destination'],
+                           settings['source']).startswith('..')
+    except:
```

Comment:
Could you specify here the type of exception that must be catched and add a comment ?